spiffe/expect.go

```diff
@@ -36,7 +36,7 @@ func ExpectPeers(expectedIDs ...string) ExpectPeerFunc {
 	}
 	return func(peerID string, _ [][]*x509.Certificate) error {
 		if _, ok := m[peerID]; !ok {
-			return fmt.Errorf("unexpected peer ID %q", peerID)
+			return fmt.Errorf("unexpected peer ID %q. We were expecting one of %q", peerID, expectedIDs)
```

Comment:
This list could be arbitrarily long.... maybe it's worth only showing the actual IDs if the number of IDs is under some threshold?

Reply:
Eh. I'm ok with printing it.